frontend/debian/cuttlefish-orchestration.cuttlefish-host_orchestrator.default

```diff
@@ -1,22 +1,13 @@
 # defaults for cuttlefish-host_orchestrator
 
-# The network address to listen on for both HTTP and HTTPS.
+# The network address to listen on.
 # Defaults to localhost only.
 orchestrator_listen_address=0.0.0.0
 #
 # The port on which the orchestrator server should listen on for plain HTTP.
-# Defaults to 2080.
+# Defaults to 2081.
 # orchestrator_http_port=
 #
-# The port on which the orchestrator server should listen on for HTTPS.
-# The orchestrator won't server over HTTPS if the port is not set.
-orchestrator_https_port=2443
```

Comment:
Random idea: Couldn't we get rid of `operator_https_port` as well?

Reply:
yes, `ssl` would be now a responsibility of `nginx`, this would be done in future PRs.

Reply:
SG.